src/components/Lane/LaneHeader.js

```diff
@@ -6,12 +6,12 @@ import {Title, LaneHeader, RightContent } from 'styles/Base'
 import LaneMenu from './LaneHeader/LaneMenu'
 
 const LaneHeaderComponent = ({
-  updateTitle, canAddLanes, onDelete, onDoubleClick, inlineEditTitle, label, title, titleStyle, labelStyle, t
+  updateTitle, canAddLanes, onDelete, onDoubleClick, inlineEditTitle, label, title, titleStyle, labelStyle, t, draggable, laneDraggable
 }) => {
 
   return (
     <LaneHeader onDoubleClick={onDoubleClick}>
-      <Title style={titleStyle}>
+      <Title style={{cursor: draggable && laneDraggable ? 'grab' : 'auto', ...titleStyle}}>
```

Comment:
👍 

As we work on components customization through dependency injection it is better to not hardcode styles but use props for `styled-components`. For example:

```
<Title laneDraggable={laneDraggable} style={titleStyle}>
```
```
export const Title = styled.span`
	  font-weight: bold;
	  font-size: 15px;
	  line-height: 18px;
       ${props => props.langeDraggable && css`
           cursor: grab;
        `}
          cursor: auto;
	  width: 70%;
	`
```